modules.d/91crypt-gpg/module-setup.sh

```diff
@@ -5,6 +5,13 @@
 check() {
     require_binaries gpg || return 1
 
+    if sc_requested; then
+        if ! sc_supported; then
+            dwarning "crypt-gpg: GnuPG >= 2.1 with scdaemon and libusb required for ccid smartcard support"
+            return 1
+        fi
```

Comment:
Thanks for the info. I agree and it's done in 0e1b686b8367925b21c76b9d9e36bf7d2f6b89ac